pkg/streams/streams_test.go

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package streams
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestValidateStreamNameFlagReturnsError_EmptyOrNullValue(t *testing.T) {
+
+	invalidStreamName := ""
+
+	loginId, err := validateStreamName(invalidStreamName)
+
+	assert.NotNil(t, err)
+	assert.Empty(t, loginId)
+}
```

Comment:
This is fine, but we could also have a test in the streamsGet_test file that tries to pass in an invalid stream name? Then we could check the error is what we expect it to be.